.github/workflows/ci.yml

```diff
@@ -21,9 +21,9 @@ jobs:
       - name: flake8 Lint
```

Comment:
We can remove flake8 from here..as we are replacing it with pre-commit.
This file can be updated similar to this - https://github.com/cloud-bulldozer/kraken/blob/master/.github/workflows/ci.ym